jmdict.go

```diff
@@ -63,13 +63,23 @@ func doDisplaySenseNumberTag(headword headword, entry jmdict.JmdictEntry, meta j
 }
 
 func jmdictPublicationDate(dictionary jmdict.Jmdict) string {
+	if len(dictionary.Entries) == 0 {
+		return "unknown"
```

Comment:
nit: too much repetition of "unknown", just nest conditionals so that success case returns `jmdictDate` and everything else returns "uknown"

Reply:
This was indeed a bit messy. I've made it a little cleaner.